.github/workflows/trivy-image-scan.yml

```diff
@@ -17,7 +17,11 @@ on:
         required: false
         type: string
         description: Only allowed for BUILD image, MUST NOT be used with DEPLOY image. Full relative path to trivy ignore file.
-    
+      DRY_RUN:
+        required: false
+        default: false
+        type: boolean
+        description: Flag to download an artifact image (named as ${SCAN_NAME}.tar) and scan
```

Comment:
Description doesn't seem to match the usage and contradicts other definitions of the `DRY_RUN` input in other modified workflows

Reply:
Renamed to IMAGE_FROM_ARTIFACT and updated description